Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,13 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const uniques = [];
+  arr.forEach(element => {
+    if (uniques.find(unique => unique === element) === undefined) {
```

Comment:
This could be simplified to:

```js
if (!uniques.find(unique => unique === element)) {
```

Even simpler:

```js
if (!uniques.includes(element)) {
```